Stack/Opc.Ua.Core/Stack/Server/ServerBase.cs

```diff
@@ -1537,6 +1537,14 @@ public void ScheduleIncomingRequest(IEndpointIncomingRequest request)
                         Utils.LogTrace("Too many operations. Total: {0} Active: {1}",
                             totalThreadCount, activeThreadCount);
                     }
+                    // now, check if the server is stopped
+                    else if (m_stopped)
+                    {
```

Comment:
very unlikely case, but stopped should be checked first, otherwise if the request queue was already full, stopped is never returned

Reply:
updated